server/src/answer/flags.js

```diff
@@ -0,0 +1,45 @@
+const getFlagsQuery = `
+	query getFlags($answerId: ID!) {
+		Answer(id: $answerId) {
+			node {
+				flags {
+					id
+				}
+			}
+		}
+	}
+`
+
+const deleteFlagQuery = `
+	mutation deleteFlag($flagId: ID!) {
+		deleteFlag(id: $flagId) {
+			id
+		}
+	}
+`
+
+const getFlags = (graphcool, answer) => {
+  return new Promise((resolve, reject) => {
+    graphcool
+      .request(getFlagsQuery, { answerId: answer.answerId })
+      .then(data => {
+        resolve(data.Answer.node.flags)
+      })
+  })
+}
+
+const deleteFlags = (graphcool, answer) => {
```

Comment:
Suggestion:

With async/await you could write more concise and readable code. What do you think about it ?

```js
const deleteFlags = async (graphcool, answer) => {
  const flags = await getFlags(graphcool, answer)
  await Promise.all(flags.map(({ id: flagId }) => {
    return graphcool.request(deleteFlagQuery, { flagId })
  }))
}
```

Reply:
Yes, you are right. I still have to wrap my head around async/await !